src/main/java/LadderGame.java

```diff
@@ -7,49 +9,48 @@
     public LadderGame(int peopleCount, int ladderHeight) {
         this.peopleCount = peopleCount;
         this.ladderHeight = ladderHeight;
-        this.ladderBoard = new char[ladderHeight][peopleCount + 2];
+        this.ladderBoard = new char[ladderHeight][peopleCount * 2 - 1];
         initBoard();
     }
 
-    public void start() {
-        makeRandomLadder();
-        showLadder();
+    private void initBoard() {
+        for (int i = 0; i < ladderHeight; i++) {
+            putInitWidth(i);
+        }
     }
 
-    private void showLadder() {
-        for (char[] chars : ladderBoard) {
-            for (char aChar : chars) {
-                System.out.print(aChar);
-            }
-            System.out.println();
+    private void putInitWidth(int height) {
+        for (int j = 0; j < peopleCount * 2 - 1; j += 2) {
+            ladderBoard[height][j] = '|';
         }
     }
 
-    private void initBoard() {
-        for (int i = 0; i < ladderHeight; i++) {
-            for (int j = 0; j < peopleCount + 2; j+=2) {
-                ladderBoard[i][j] = '|';
-            }
-        }
+    public void start() {
+        makeRandomLadder();
+        PrintView.showLadder(ladderBoard);
     }
 
     private void makeRandomLadder() {
         for (int i = 0; i < ladderHeight; i++) {
-            for (int j = 1; j < peopleCount + 2; j+=2) {
-                if (isPutLine()) {
-                    ladderBoard[i][j] = '-';
-                    continue;
-                }
-                ladderBoard[i][j] = ' ';
-            }
+            makeRow(i);
         }
     }
 
-    private boolean isPutLine() {
-        return makeRandomValue() > 5;
+    private void makeRow(int i) {
+        for (int j = 1; j < peopleCount * 2 - 1; j += 2) {
+            putRandomLine(i, j);
+        }
     }
 
-    private int makeRandomValue() {
-        return (int)(Math.random() * 10 + 1);
+    private void putRandomLine(int i, int j) {
+        if (isPutLine()) {
+            ladderBoard[i][j] = '-';
+            return;
+        }
+        ladderBoard[i][j] = ' ';
+    }
+
+    private boolean isPutLine() {
```

Comment:
```suggestion
    private boolean putLine() {
```
불리언이라고 is를 꼭 쓸 필요는 없습니다. 영어 문장으로 너무 어색해요.